File: pyjl/tests/tests_new/yield_from_test.py
def generator1():
    for i in range(3):
        yield i

def generator():
    yield from generator1()
    yield from generator2()
    # print(map)

def generator2():
    for j in range(3, 5):
        yield j

if __name__=="__main__":
    arr = []
    for i in generator():
        arr.append(i)
    assert arr == [0,1,2,3,4]

File: pyjl/tests/tests_new-py2many/yield_from_test.jl
using ResumableFunctions
 @resumable function generator1()
for i in (0:2)
@yield i;
end
end

function generator()
generator1();
generator2();
end

 @resumable function generator2()
for j in (3:4)
@yield j;
end
end

function main()
arr = []
for i in generator()
push!(arr, i);
end
@assert(arr == [0, 1, 2, 3, 4])
end

main()
